### PR TITLE
one more dataset to illustrate variety of applications

### PR DESCRIPTION
**Pull request by LinguList**
this dataset theoretically contains partial cogantes, but no IPA at all, since all values are given in Chinese characters. This shows how great the variation is among data we can find. The dataset is valuable for tree calcs, though, as it offers basic vocabulary lists for some 20 Chinese dialects. So it is useful to have it there (and it also has proto-forms, which means it can be used for semantic reconstruction to test it, etc.)

In general, we are probably close to having most points covered (not implemented, but we have examples for them). 

In the graphic below, I modify one point, namely showing now, what values can be computed, and what can be achieved at by using available annotation tools:

rect5342-9-2

It seems that apart from Partial cognates, we have quite some range covered now.

### CODE REVIEW

File: datasets/cbvl/__init__.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        SOURCE,
+                        p2c.get(val, val)
+                        ))
+                dataset.cognates += [[
+                    '{0}-{1}'.format(SOURCE, idx), ds.name, val,
+                    '-'.join([slug(line[0]), str(p2c.get(val, val))]),
+                    '', 'expert', SOURCE, '', '', '']]
+                idx += 1
+
+        dataset.write_cognates()
```
Comment (xrotwang) [inline datasets/cbvl/__init__.py]: This line is not necessary, since this is done by `lexibank.cli.cldf` after the individual `cldf` method has been run.

Comment (LinguList) [inline datasets/cbvl/__init__.py]: you mean, datase.write_cognates() ?

**Comment by xrotwang** [comment]
Should the arrow between "Cognate-Sets" and "Proto-Forms" point in the opposite direction?

File: datasets/cbvl/__init__.py (hunk 1 above)
Comment (xrotwang) [inline datasets/cbvl/__init__.py]: Yes, see https://github.com/glottobank/lexibank-data/blob/master/pylexibank/cli.py#L109

**Comment by LinguList** [comment]
well, I guess I should change the "requires", as I meant: "makes possible" so, A makes possible B, and cognate sets makes possible Proto-Forms. because, without cognate sets, we can't have a proto-form, I'd say, at least according to strict classical doctrine...

**Comment by xrotwang** [comment]
yes, that would be more transparent for me.

**Comment by LinguList** [comment]
I'll just add some more lines to the semitic data, as it contains bad characters, and then I'll remove the lines which are not needed...

**Comment by xrotwang** [comment]
I've already merged, so you may have to create a new PR

**Pull request merged**